src/injected/visualization/tab-stops-formatter.ts

```diff
@@ -46,7 +46,7 @@ export class TabStopsFormatter implements Formatter {
             erroredCircle: {
                 stroke: '#E81123',
                 strokeWidth: '2',
-                fill: '#ffffff',
+                fill: 'transparent',
```

Comment:
Have we checked with Fer if displaying these circles as transparent is okay, as it doesn't match the figma? 

Overall, this approach seems like it can get us into some weird states. For example, when smoke testing for a minute I noticed that at times we can get into a state where the focus circle has a tab index in it and where other tab index labels disappear suddenly. See the screenshot below, in which focus is on item 4 and, prior to the last tab, both of the error circles had tab index labels which have now disappeared. 
[Screenshot of 4 tabbed elements, 2 red error circles, one focus circle, and one regular circle.]
<img width="64" alt="tabstopstesting" src="https://user-images.githubusercontent.com/16010855/157279471-a98c09e4-7646-4ea0-8988-764b2b89c8c1.png">

Reply:
Discussed offline.